### ISSUE TITLE
No way to remove the default layout from a display

### ISSUE THREAD

**Issue by andyblackham** [xibosignage/xibo#199]
Once you set a default layout in the displays window, there is no way to remove the default, so it returns back to the black screen with the Spring logo in the corner.

**Comment by dasgarner** [comment]
Each display must always have a default layout, which is shown when there are no other layouts scheduled or valid.

If the splash screen is showing it probably means your default layout is invalid - so the only thing the player can do is revert to splash screen.

**Comment by nagyrobi** [comment]
The default layout is invalid if upgrading from 1.7.0, also invalid when upgrading from 1.6.x even if the layout is upgraded manually. The default layout can be corrected by re-specifying the black background color.

But better - everyone should perhaps create his/her own default layout from scratch.

**Comment by andyblackham** [comment]
I got around it by deleting the layout that was set as the default and they
returned the client to the sprint logo black screen

But, the words Each display must always suggest that its mandatory or worthy
of an administrative alert that this needs setting / changing.

andy


From:  Dan Garner <notifications@github.com>
Reply-To:  xibosignage/xibo
<reply+00a686d51c0d82e915728b2f56057dadff1b68ece15ff74892cf0000000110f1b7449
2a169ce03674d4c@reply.github.com>
Date:  Tuesday, 10 February 2015 12:10
To:  xibosignage/xibo <xibo@noreply.github.com>
Cc:  Andy Blackham <andy.blackham@ntlworld.com>
Subject:  Re: [xibo] No way to remove the default layout from a display
(#199)


Each display must always have a default layout, which is shown when there
are no other layouts scheduled or valid.

If the splash screen is showing it probably means your default layout is
invalid - so the only thing the player can do is revert to splash screen.

—
Reply to this email directly or view it on GitHub
<https://github.com/xibosignage/xibo/issues/199#issuecomment-73689482> .

**Comment by dasgarner** [comment]
Let me rephrase then - each display *should* have a default layout, otherwise the splash screen will be displayed when nothing else is scheduled.

The default layout concept is designed to provide a sensible way to show something specific when nothing else is scheduled to be shown.

You are totally correct that a "please configure you default layout" section of the getting started guide would be useful. We are working on that at the moment.

**Comment by andyblackham** [comment]
are there any specific areas of documentation you'd like help in?

**Comment by dasgarner** [comment]
The structure is all there as is some basic content, it just needs fleshing out a little in all areas. For example, there is information on the Display Management page, but not sufficient to answer your question - if there were, that would be great (same for other areas).

I think generally speaking, if you are using the system and hit something you can't answer - please ask as you did - but then that subject is prime candidate to expand the manual.

The manual is all in markdown format xibosignage/xibo-manual - in the "source" folder and it compiled to HTML using generate.php. Editing the source markdown is sufficient to get changes into the manual.

**Comment by dasgarner** [comment]
I am closing this issue because we have moved all discussion and support to [community.xibo.org.uk](http://community.xibo.org.uk) and would like to tidy up GitHub so we can use this as a **bug tracker** only. If you are still having a problem, please repost on the [community site](http://community.xibo.org.uk) and please update your bookmarks for future questions!
